src/App.tsx

```diff
@@ -1,14 +1,37 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
 import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string;
+};
+
+interface Event {
+  target: {
+    value: string;
+  };
+}
```

Comment:
You shoud use `React.ChangeEvent<HTMLInputElement>` type instead of this custom interface👆